README.md

```diff
@@ -39,6 +39,12 @@ nvm use lts/hydrogen
   * Install [kubectl](https://kubernetes.io/docs/tasks/tools/)
   * Install [k9s](https://k9scli.io/)
 
+## Clone the Hedera Repository and navigate to the project directory:
+```
+git clone https://github.com/hashgraph/solo.git
+cd solo
+```
+
```

Comment:
actually, we don't want this on the readme.  we want them to just do the `npm install -g @hashgraph/solo`

we need to remove these two:
```
Run npm clean install or npm ci
Run npm run build
```

this belongs in the developer area of the hugo docs published pages.